core/messages/en.json

```diff
@@ -286,7 +286,8 @@
       "confirmPasswordValidationMessage": "Entered passwords are mismatched. Please try again.",
       "newPasswordValidationMessage": "New password must be different from the current password or/and match confirm password.",
       "notEmptyMessage": "Field should not be empty",
-      "successMessage": "Password has been updated successfully!"
+      "successMessage": "Password has been updated successfully!",
+      "confirmChangePassword": "Your password has been successfully updated."
```

Comment:
Do we need both translations?

Reply:
@bc-alexsaiannyi I've double-checked this message in Figma and fixed it

Reply:
thanks!